packages/babel-traverse/src/context.js

```diff
@@ -105,6 +105,9 @@ export default class TraversalContext {
       path.resync();
       path.pushContext(this);
 
+      // this path is no longer belongs to the tree
+      if (path.key === null) continue;
```

Comment:
@amasad It was set to null here:
https://github.com/babel/babel/blob/master/packages/babel-traverse/src/path/context.js#L165

And we can reproduce it with two steps:

1. insert a path into a upper level queue (VariableDeclarationStatement was inserted after the FunctionDeclarationStatement)
2. replace the path's parent path (FunctionDeclarationStatement was replaced with a CallExpression)

But I can't figure out why the container of inserted path is not modified after the _containerInsert operation. Any idea?